orca/scripts/build_services.sh

```diff
@@ -278,6 +278,38 @@ function build_service_search {
 	write 1 "        - \"9300:9300\""
 }
 
+function build_service_teleport_agent_test {
+	docker_build teleport-agent-test
+
+	write 1 "${SERVICE_NAME}:"
+	write 1 "    container_name: ${SERVICE_NAME}"
+	write 1 "    hostname: ${SERVICE_HOST}"
+	write 1 "    image: teleport-agent-test:${VERSION}"
+	write 1 "    volumes:"
+	write 1 "        - /opt/liferay/teleport/agent-test:/agent-test"
+}
+
+function build_service_teleport_proxy {
+	docker_build teleport-proxy
+
+	write 1 "${SERVICE_NAME}:"
+	write 1 "    container_name: ${SERVICE_NAME}"
+	write 1 "    environment:"
+	write 1 "        - GITHUB_ID=$(query_github_creds ".github.id")"
+	write 1 "        - GITHUB_SECRET=$(query_github_creds ".github.secret")"
```

Comment:
Later (once we decide to use teleport) this should come from vault, it should not be stored in the configuration. But we don't need to implement it now, we can read it from an environment variable like here: https://github.com/liferay/liferay-docker/blob/master/orca/scripts/build_services.sh#L66